src/widgets/DropdownButton.js

```diff
@@ -264,14 +264,19 @@ define(function (require, exports, module) {
             posTop = Math.max(0, toggleOffset.top - $dropdown.height() - 4);
         }
 
+        // Take in consideration the scrollbar to prevent the edit icon to move down. See #10963
```

Comment:
Maybe we shouldn't mention the edit icon here because DropdownButton is far more versatile.
I'd go for something along the lines of `Take the scrollbar into consideration so we don't get unintented line breaks (see #10963)`